docs/src/further_topics/user_experience/ux_past_discussions/ux_discussion_02-05-25.rst

```diff
@@ -0,0 +1,126 @@
+Iris UX Discussion 02-05-25
+###########################
+
+Community
+*********
+
+Low User Interaction
+=====================
+
+* Iris has always been a bit difficult to discover, as it covers quite a niche area.
+  We think that with the more popular and general-fit xarray, not only are we hard to discover,
+  but our ideal users might be finding xarray, calling it "good enough", and not putting in the
+  extra effort to find the better fitting packages.
+
+* It might also be that Iris' user base is largely Met Office employees, who would rely on
+  support and Viva Engage. We *DO NOT* think that we should put more focus onto Met Office
+  employees, as that would skew the balance and dissuade further users from joining us.
```

Comment:
I am comfortable sharing things like this, but it needs to make sense to all readers. Phrases like `UK Met Office` and `internal message boards`